python/graphstorm/model/node_gnn.py

```diff
@@ -316,8 +317,11 @@ def node_mini_batch_gnn_predict(model, loader, return_proba=True, return_label=F
                               iter_l, max_num_batch, time.time() - iter_start)
 
     model.train()
-    for ntype, ntype_pred in preds.items():
-        preds[ntype] = th.cat(ntype_pred)
+    # MFG for DGL 2.0.0+ return all node and edge type
```

Comment:
Can you also check ``node_mini_batch_predict``?

Reply:
I manually check that function and do not require any other change because it only requires decoder. Only the GNN related predict function needs some modifications.